content/swagger/resources/attributes/definitions/attribute.yaml

```diff
@@ -134,6 +134,10 @@ properties:
     description: Reference entity code when the attribute type is `akeneo_reference_entity` or `akeneo_reference_entity_collection` OR Asset family code when the attribute type is `pim_catalog_asset_collection`
     x-validation-rules: Required only for `akeneo_reference_entity` or `akeneo_reference_entity_collection` or `pim_catalog_asset_collection` attribute types
     x-warning: Only for `akeneo_reference_entity` or `akeneo_reference_entity_collection` or `pim_catalog_asset_collection` attribute types
+  default_value:
+    type: boolean
+    description: Default value for a Yes/No attribute, applied when creating a new product or product model (only available in Serenity)
```

Comment:
> (only available in Serenity)

we will have to change this when the 5.0 is released (smth like available as of v5.0), I'm not sure how to handle that...

Reply:
I already scheduled to spend some time reviewing the API doc to prepare the 5.0 release. No worries ;)